I_have_no_vignette.py

```diff
@@ -0,0 +1,34 @@
+import random
+from datetime import date, timedelta, datetime
+import calendar
+
+
+def check_monday(date):
+    print(f'The random date is : {date.date()}')
+    day = calendar.day_name[date.weekday()]
+    if day == "Monday":
+        print("אין לי ויניגרט!")
+
+
+def random_date(date1, date2):         #The left date is bigger
+    dates_bet = (date2 - date1).days
+    date_between = random.randrange(dates_bet)
```

Comment:
Change the names of date_between and dates_bet.
Its difficult to understand the differences between them, and morover, what do they do.